### ISSUE TITLE
402 Assignments

### ISSUE THREAD

**Issue by yshin4** [dondi/GRNsight#554]
GRNsight: Node Coloring & Alternate Graph Layout

- [ ] Week 03: 2017-01-22 Project Proposal Presentations
* http://bjohnson.lmu.build/cmsi402web/proposal.html

- [ ] Week 05: 2017-02-05 - Requirments Document (SRS)
* http://bjohnson.lmu.build/cmsi402web/analysispg.html
* http://bjohnson.lmu.build/cmsi402web/requirem.html

- [ ] Week 07: 2017-02-19 - Written Status Reports in SDF

- [ ] Week 08: 2017-02-26 - Software Development Plan or Software Design Description (Initial)
* http://bjohnson.lmu.build/cmsi402web/sdevplan.html
* http://bjohnson.lmu.build/cmsi402web/sddesign.html
- [ ] Week 08: 2017-02-26 - Oral Status Reports

- [ ] Week 11: 2017-03-19 - Software Development Plan or Software Design Description (Re-submit)
- [ ] Week 11: 2017-03-19 - Written Status Reports in SDF

- [ ] Week 13: 2017-04-02 - Requirments Document (Re-submit)
- [ ] Week 13: 2017-04-02 - Written Status Reports in SDF

- [ ] Week 14: 2017-04-09 - Preliminary Version of Poster

**Comment by kdahlquist** [comment]
Just so I am clear, each of these check boxes is a joint product of @yshin4 and @eileenchoe?

I was actually thinking that each item could be a separate issue so that it could be closed when finished.  This is OK, too, as long as it serves as a useful reminder as to what needs to be done.

**Comment by yshin4** [comment]
@kdahlquist Yes, we will be submitting one as a group. I can make a separate one if @eileenchoe wants to do it that way.

**Comment by eileenchoe** [comment]
Ok @yshin4 and I decided that we'll break this up into issues based on deliverable requirements for 402.

**Comment by eileenchoe** [comment]
Final Poster for CMSI 402 is uploaded [here](https://github.com/dondi/GRNsight/blob/master/documents/SDF/CMSI_402/spring_2018/ChoeShinCMSI402_4:22.pptx). @kdahlquist @dondi can we review this at the meeting tomorrow?

CMSI402 presentations are Monday, April 30th, at 4:30 PM. @yshin4 and I will be presenting at the poster session.

**Comment by kdahlquist** [comment]
After the CMSI 402 presentations/posters this afternoon @yshin4  or @eileenchoe should have the pleasure of closing this one!
